types/apple-mapkit-js-browser/index.d.ts

```diff
@@ -60,6 +60,11 @@ declare namespace mapkit {
          * labels, controls, directions, and other text.
          */
         language?: string | undefined;
+
+        /**
+         * An array of strings specifying the MapKit JS libraries to load at initialization. 
+         */
+        libraries?: string[] | undefined;
```

Comment:
If this array is never modified, then it should be marked `readonly`:

```suggestion
        libraries?: readonly string[] | undefined;
```